libs/librepcb/common/fileio/filepath.cpp

```diff
@@ -265,10 +265,11 @@ QString FilePath::cleanFileName(const QString& userInput, CleanFileNameOptions o
     // replace spaces with underscore
     if (options.testFlag(CleanFileNameOption::ReplaceSpaces)) ret.replace(' ', '_');
     // remove all invalid characters
-    QString validChars("_\\-.0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz");
+    QString validChars(" _\\-.0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz");
```

Comment:
Now I got it: The backslash is required to escape the minus sign because it is a reserved regex character ;)

Nevertheless I have created a separate PR to fix and improve this function: #160